### PR TITLE
open-cluster-management/backlog#1580 --new guide edits.

### PR DESCRIPTION
**Pull request by swopebe**
@cadawson quick edits, can you take a look?

Thanks

### CODE REVIEW

**Comment by swopebe** [comment]
a link, I mean, not all the known issues

**Comment by swopebe** [comment]
@cadawson poke

File: _pages/new_guide_setup.md [hunk 1]
```diff
@@ -3,20 +3,21 @@ permalink: /new_documentation
...
 * The `open-cluster-management` repository
 * Red Hat VPN
 * Pantheon
+* GitLab
 
-**Important**: Do you really need a new guide? See the doucmentation architect to discuss this.
+**Important**: Do you really need a new guide? See the documentation architect to discuss placement of new content.
 
-To create a new guide in the product documentation, you must create a new folder in your repository, and a new book in Pantheon. When you create a new folder, you must also create a `master.adoc` and `docinfo.xml` file for your new folder.  
+To create a new guide in the product documentation, you must create a new folder in your repository, and a new book in Pantheon. When you create a new folder, you must also create a `master.adoc` and `docinfo.xml` file for your new folder. See the repo for examples.
```
Comment (dockerymick) [inline _pages/new_guide_setup.md]: I suggest to spell out repo

File: _pages/new_guide_setup.md [hunk 2]
```diff
@@ -53,6 +54,7 @@ To create a new guide in the product documentation, you must create a new folder
    ```
 
 **Create a new folder from your text editor**
+(VS Code or Atom is recommended to preview, prevent errors, work in collaboration with dev.)
```
Comment (dockerymick) [inline _pages/new_guide_setup.md]: I would type out VS ----> Visual Studio

File: _pages/new_guide_setup.md [hunk 3]
```diff
@@ -97,6 +101,18 @@ You must complete the _Set up new title_ form to create a new book. Log in to Pa
    8. You can add multiple new titles by selecting the **Add additional title** button. Otherwise, select **Submit** when the form is complete. 
    9. Run a manual build to pick up the new book. For more information, see [Build and publish](arch_builds.md#build-and-publish).
 
-## Renaming (need this because we have to do this)
+## Renaming 
+
+Only rename a folder/guide when absolutely needed as a result of product change.
+
+   1. Rename the folder in the source, as Git should match the URL. (Git remains the source of truth.)
```
Comment (dockerymick) [inline _pages/new_guide_setup.md]: change should match to must match

File: _pages/new_guide_setup.md [hunk 4]
```diff
@@ -97,6 +101,18 @@ You must complete the _Set up new title_ form to create a new book. Log in to Pa
...
+   2. Rename the guide in the `docinfo.xml` file.
+   3. Rename the title of the `master.adoc`.
+   4. Using find/replace, change all links that are impacted.
+   5. Log in to Pantheon.
+   6. Click **Edit title**.
+   7. Change the name and the URL fragment. Keep URL fragment
```
Comment (cadawson) [inline _pages/new_guide_setup.md]: Should this be either change the URL fragment or keep the URL fragment? Is something missing?

**Review by dockerymick — COMMENTED** [review]
Made a few comments, thanks for the changes Brandi!

**Review by cadawson — APPROVED** [review]
Looks good. Thank you! I left a note about a confusing URL fragment piece.

File: _pages/new_guide_setup.md (hunk 1 above)
Comment (swopebe) [reply _pages/new_guide_setup.md]: done

File: _pages/new_guide_setup.md (hunk 3 above)
Comment (swopebe) [reply _pages/new_guide_setup.md]: changed

File: _pages/new_guide_setup.md (hunk 2 above)
Comment (swopebe) [reply _pages/new_guide_setup.md]: I think internally people will know this

File: _pages/new_guide_setup.md (hunk 4 above)
Comment (swopebe) [reply _pages/new_guide_setup.md]: changed

**Pull request merged**